tests/simpleKVBC/TesterReplica/setup.cpp

```diff
@@ -156,6 +156,10 @@ std::unique_ptr<TestSetup> TestSetup::ParseArgs(int argc, char** argv) {
           replicaConfig.batchFlushPeriod = batchFlushPeriod;
           break;
         }
+        case 'u': {
+          replicaConfig.blockAccumulation = true;
+          break;
+        }
```

Comment:
I think this isn't necessary any more.

Reply:
This is necessary since we don't want to enable block accumulation for all test cases.

Reply:
Sorry, I see now how it's being used in ReplicaImp.